api/call.go

```diff
@@ -42,8 +41,9 @@ type request struct {
 }
 
 type answer struct {
-	Error  string      `json:"error,omitempty"`
-	Result interface{} `json:"result,omitempty"`
+	Error   string      `json:"error,omitempty"`
+	Result  interface{} `json:"result,omitempty"`
+	TraceID string      `json:"traceID,omitempty"`
```

Comment:
trace id should generate on start of request lifeline, consider to move TraceID generation on client (request) side